### ISSUE TITLE
codt - statistic 

### ISSUE THREAD

**Issue by ARX778** [openplanet-nl/tmio#64]
... on trackmania.io: It would be nice if you had the possibility to make the queries yourself via form: For example: What is my average in the last 6 months including reruns or exclusive.Otherwise the statistic makes no sense, because it hardly changes after about 6 months and you cannot observe the development.

**Comment by codecat** [comment]
True, perhaps the average should already be "in the last year" or something.

Though I'm not sure we need to have it be configurable, it wouldn't really fit in the UI and it'd be kinda inconsistent.

**Comment by ARX778** [comment]
Whether by random or intention: If one participates in the qualification but not in the race, the rank 64 for the race is now not recorded, so that the "Average div rank" is correct. Previously, the title "Average div rank" was false. I hope this is not a random change in the data source but rather a bug fix.

**Comment by davidbmaier** [comment]
Closing this to move further discussion into #132 - I'll hopefully take a closer look at that soon, and it should cover the same use cases as this one
